scripts/reverseproxy-facts.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# vim: sw=2 ts=2 ft=python
```

Comment:
I don't think so

Reply:
Yep, he forgot `et`

Reply:
Also, the modeline is compatible with vi, sou you can use `vi:` instead of `vim:`

Reply:
And shouldn't there be a trailing colon?